app/client/ui/DocumentSettings.ts

```diff
@@ -162,7 +162,7 @@ export class DocSettingsPage extends Disposable {
         }) : null,
       ]),
 
-      dom.create(AdminSection, t('API'), [
+      dom.create(cssAdminSection, t('API'), [
```

Comment:
Can you take a look at AdminPanel.ts ? It also uses those styles, maybe we can just update the base component, without wrapping it into `cssAdminSection`